Plots/Boxplots/NCL_box_2.py

```diff
@@ -0,0 +1,95 @@
+"""
+NCL_box_1.py
+===============
```

Comment:
```suggestion
NCL_box_1.py
============
```

Reply:
I see. Does that mean I should be on the main branch when I do git checkout -b newbranch?

Reply:
Yup exactly that!

Reply:
Got it. Should I try and delete those files from this branch? Also by length of the range markers, do you mean the length of the whisker? (I'm assuming using the whis argument)

Reply:
`whis` is not actually the argument you want. I think you'll actually want to modify `cap`